README.md

```diff
@@ -1,19 +1,30 @@
 # Conjur API for .NET
 
-This is a *Draft* implementation of the .NET API for [V5 Conjur](https://developer.conjur.net/).
-This implementation includes an example that shows how to:
+Programmatic .NET access to the [Conjur API](https://docs.conjur.org/latest/en/Content/Developer/lp_REST_API.htm?tocpath=Developer%7CREST%C2%A0APIs%7C_____0#rest-api) (for both Conjur OSS and Enterprise/DAP versions).
```

Comment:
Just change this to `[Conjur](https://conjur.org)` - I think that doc link you have here might be liable to break at a later date.